.github/workflows/cd.yaml

```diff
@@ -63,7 +83,7 @@ jobs:
             # repair wheel
             cd $HOME
             pip wheel -w broken-dist/ .
-            auditwheel repair -w dist/ --plat manylinux_2_27_x86_64 broken-dist/*.whl
+            auditwheel repair -w dist/ --plat manylinux_2_31_x86_64 broken-dist/*.whl
```

Comment:
This part did not originally come from this PR, but is it even needed? As far as I understand, the wheel is already being built in the Dockerfile, so we should not need to build it here again. In any case, repairing the wheel with `manylinux_2_31` would make even the Python 3.8 and 3.9 wheels incompatible with `manylinux_2_27` again